netboot/main.go

```diff
@@ -150,7 +156,7 @@ func boot(ifname string, dhcp dhcpFunc) error {
 		}
 		debug("DHCP: network configuration: %+v", netconf)
 		if !*dryRun {
-			log.Printf("DHCP: configuring network interface %s", ifname)
+			log.Printf("DHCP: configuring network interface %s with %v", ifname, netconf)
```

Comment:
nit: `%+v` also prints the names of the fields, it may help debugging

```suggestion
			log.Printf("DHCP: configuring network interface %s with %+v", ifname, netconf)
```

Reply:
We actually print netconf on the line above, if -d is set - no need to add more to the logs here.